### PR TITLE
Converter class integrated

### PR DESCRIPTION
**Pull request by luisdavidparra**

### CODE REVIEW

File: CONVERTER-SERVICE/src/controllers/converter.controller.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const VideoConverter = require('../converter/video.converter');
+
+module.exports = class ConverterController {
+  
+  // Executes and verifies the response of the 3 methods involved in the project
+  static convertVideo(req, res) {
+    const uploadRespond = Upload.uploadVideoMethod(req.file);
+
+    if (!uploadRespond) {
+      res.send('Insert a suppoted file');
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/controllers/converter.controller.js]: 'Insert a supported file" (you forgot the 'r')

File: CONVERTER-SERVICE/src/middlewares/multer.middleware.js [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+Av. General Inofuentes esquina Calle 20,Edificio Union № 1376, La Paz, Bolivia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
 const multer = require('multer');
```
Comment (florpadilla) [inline CONVERTER-SERVICE/src/middlewares/multer.middleware.js:14]: add comments to this

**Review by florpadilla — APPROVED** [review]
It looks good! the only comment is that maybe some comments need to be added

File: CONVERTER-SERVICE/src/controllers/converter.controller.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const Upload = require('../helpers/upload.helper');
+const Compress = require('../helpers/compress.helper');
+const VideoConverter = require('../converter/video.converter');
+
+module.exports = class ConverterController {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/controllers/converter.controller.js:17]: You should comment above the ConverterController class

File: CONVERTER-SERVICE/src/controllers/download.controller.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Av. General Inofuentes esquina Calle 20,Edificio Union № 1376, La Paz, Bolivia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+module.exports = class DownloadController {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/controllers/download.controller.js:13]: You should comment above the DownloadController class

File: CONVERTER-SERVICE/src/controllers/converter.controller.js (hunk 1 above)
Comment (luisdavidparra) [reply CONVERTER-SERVICE/src/controllers/converter.controller.js]: done

File: CONVERTER-SERVICE/src/middlewares/multer.middleware.js (hunk 2 above)
Comment (luisdavidparra) [reply CONVERTER-SERVICE/src/middlewares/multer.middleware.js:14]: done

File: CONVERTER-SERVICE/src/helpers/compress.helper.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const admzip = require('adm-zip');
+const fs = require('fs');
+
+module.exports = class Compress {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/compress.helper.js:16]: You should comment above the Compress class

File: CONVERTER-SERVICE/src/helpers/upload.helper.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+Av. General Inofuentes esquina Calle 20,Edificio Union № 1376, La Paz, Bolivia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+module.exports = class Upload {
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/helpers/upload.helper.js:13]: You should comment above the Upload class

**Review by DilanOjeda — COMMENTED** [review]
Add some comments above the classes

**Pull request merged**